python/ray/train/_internal/state/schema.py

```diff
@@ -46,9 +63,19 @@ class TrainRunInfo(BaseModel):
     datasets: List[TrainDatasetInfo] = Field(
         description="A List of dataset info for this Train run."
     )
+    run_status: RunStatusEnum = Field(
```

Comment:
for tracking error details, might be good to have a simple "status_details" field which is a string. 

In the future, while the run is running, you could put some additional details in that field.

Reply:
I see. You mean keep "run_status" and add an additional field "status_details"?

Reply:
yup